apps/backend/src/authn/okta.strategy.ts

```diff
@@ -26,50 +25,54 @@ export class OktaStrategy extends PassportStrategy(Strategy, 'okta') {
   ) {
     super(
       {
-        issuer: `https://${
-          configService.get('OKTA_DOMAIN') || 'disabled'
-        }/oauth2/default`,
-        authorizationURL: `https://${
-          configService.get('OKTA_DOMAIN') || 'disabled'
-        }/oauth2/default/v1/authorize`,
-        tokenURL: `https://${
-          configService.get('OKTA_DOMAIN') || 'disabled'
-        }/oauth2/default/v1/token`,
-        userInfoURL: `https://${
-          configService.get('OKTA_DOMAIN') || 'disabled'
-        }/oauth2/default/v1/userinfo`,
+        issuer:
+          configService.get('OKTA_ISSUER_URL') ||
+          `https://${configService.get('OKTA_DOMAIN')}` ||
+          'disabled',
+        authorizationURL:
+          configService.get('OKTA_AUTHORIZATION_URL') ||
+          `https://${configService.get('OKTA_DOMAIN') || 'disabled'}/oauth2/v1/authorize`,
```

Comment:
Are the new paths from switching the library?

Reply:
No, the paths come purely from okta; however, trying to be smart about what the url will be is not great for our own MITRE deployment and was causing problems in the sponsor environment.